### PR TITLE
Issues #125, 129, and 140

### PR DESCRIPTION
**Pull request by nataliebrotherton**
Issue #125 turn on debug log by default
- Added a flag in the options table to store whether the user has changed the debug log setting
- If it hasn't been changed, debug log is enabled on activation

Issue #129 Validate correct authorized application
- Server application has access to 15 endpoints
- Wild Apricot API will respond with a list of scopes. If there are less than 15, the user is prompted to change the application type to a server application

Issue #140 Remove admin-only fields
- Admin-only fields are separated into another list stored in the options table and displayed as `disabled` or grayed out checkboxes

### CODE REVIEW

**Comment by asirota** [comment]
I'll need to update read me file with changes. Can you also tag this as a 1.1 version please in the code?

**Comment by nataliebrotherton** [comment]
did you want to wait for the rest of the issues to be resolved? (fixing sync, customizing login page, etc)

**Comment by asirota** [comment]
> did you want to wait for the rest of the issues to be resolved? (fixing sync, customizing login page, etc)

I think so. You can keep doing to this draft PR. And I can test against it before merging.

**Comment by nataliebrotherton** [comment]
This branch is named specific to the issues that were fixed so I made a 1.1 branch and set that as the upstream for this one

**Review by asirota — APPROVED** [review]

File: src/admin-settings.php
Comment (asirota) [inline src/admin-settings.php:1]: WildApricot is spelled without the space always.

File: src/admin-settings.php [hunk 1]
```diff
@@ -618,7 +618,8 @@ public function restriction_message_sanitize($input)
      */
     public function custom_fields_print_info()
     {
-        print 'Please select the WildApricot Contact Fields that you would like to sync with your WordPress site.';
+        print 'Please select the WildApricot Contact Fields that you would like to sync with your WordPress site.<br>';
+        print 'Admin-only contact fields are displayed below the list of contact fields but are not available to sync. If you wish to sync these fields with your site please change the field settings in Wild Apricot.';
```
Comment (asirota) [inline src/admin-settings.php:622]: Wild Apricot should be WildApricot

File: src/admin-settings.php [hunk 2]
```diff
@@ -1274,10 +1298,14 @@ public function create_wa_auth_login_page()
...
         }
-        if (!WA_Integration::valid_wa_credentials()) {
+        $api_status = WA_Integration::get_api_input_status();
+        if ($api_status == WA_Integration::API_STATUS_INVALID) {
             // not valid
             echo '<p class="wap-error">Missing valid WildApricot credentials. Please enter them above.</p>';
-        } elseif ($wild_apricot_url) {
+        } elseif ($api_status == WA_Integration::API_STATUS_INCORRECT_SCOPE) {
+            // incorrect scope selected
+            echo '<p class="wap-error">Incorrect application scope. Please create a <strong>full-access Server Application</strong> on Wild Apricot.</p>';
```
Comment (asirota) [inline src/admin-settings.php:1307]: WildApricot is the correct spelling

**Review by asirota — CHANGES_REQUESTED** [review]
just a few typos on WildApricot spelling

**Comment by asirota** [comment]
> This branch is named specific to the issues that were fixed so I made a 1.1 branch and set that as the upstream for this one

Ok, this pull is set to go  against master so you probably want to change the target to 1.1

**Comment by asirota** [comment]
@nataliebrotherton  please confirm the changes I requested and make the edits.

**Review by asirota — APPROVED** [review]

**Pull request merged**